src/pocketdb/web/PocketExplorerRpc.cpp

```diff
@@ -221,6 +221,45 @@ namespace PocketWeb::PocketWebRpc
         return addressInfo;
     }
 
+    UniValue GetBalanceHistory(const JSONRPCRequest& request)
+    {
+        if (request.fHelp)
+            throw std::runtime_error(
+                "getbalancehistory \"address\" topHeight count\n"
```

Comment:
What does topheight refer to?  Block height or starting transaction number?

Reply:
block height